examples/deployment/modal-example/client/javascript/src/app.js

```diff
@@ -93,23 +93,12 @@ class ChatbotClient {
   }
 
   /**
-   * Set up the RTVI client and Daily transport
+   * Set up the Pipecat client and Daily transport
    */
   async initializeClientAndTransport() {
-    // Initialize the RTVI client with a DailyTransport and our configuration
-    this.rtviClient = new RTVIClient({
-      transport: new DailyTransport(),
-      params: {
-        // REPLACE WITH YOUR MODAL URL ENDPOINT
-        baseUrl:
-          'https://<Modal workspace>--pipecat-modal-bot-launcher.modal.run',
-        endpoints: {
-          connect: '/connect',
-        },
-        requestData: {
-          bot_name: 'openai',
-        },
-      },
+    // Initialize the Pipecat client with a DailyTransport and our configuration
+    this.pcClient = new PipecatClient({
+      transport: DailyTransport.create(),
```

Comment:
for most clients, this is the primary change:
- transport now takes a factory function
- `params` is not a thing, but see `connect()` where you can still pass in the endpoint for getting credentials.